src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntity.java

```diff
@@ -1183,4 +1185,9 @@ public Timing getTimingsHandler() {
         }
         return this.timing;
     }
+
+    @Override
+    public boolean hasGravity() {
+        return this.get(Keys.ENTITY_HAS_GRAVITY).get();
+    }
```

Comment:
Sorry, I was meaning similar to the other Entity shortcuts, where the API default-implements `getValue(Key)`, and then the implementation overrides with `new SpongeValue(Key, default, value)`.